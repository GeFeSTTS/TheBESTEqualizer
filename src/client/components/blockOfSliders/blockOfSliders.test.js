import React from 'react';
import Adapter from 'enzyme-adapter-react-16';
import { shallow, configure } from 'enzyme';
import toJson from 'enzyme-to-json';
import BlockOfSliders from './blockOfSliders';

configure({ adapter: new Adapter() });

describe('<BlockOfSliders />', () => {
  it('matches snapshot', () => {
    const props = {
      name: 'Delay',
      effects: {
        feedback: {
          value: 0,
          minValue: 0,
          maxValue: 1,
          step: 0.01,
        },
        time: {
          value: 0,
          minValue: 0,
          maxValue: 5,
          step: 0.001,
        },
        mix: {
          value: 0,
          minValue: 0,
          maxValue: 1,
          step: 0.001,
        },
      },
    };
    const wrapper = shallow(<BlockOfSliders {...props} />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
