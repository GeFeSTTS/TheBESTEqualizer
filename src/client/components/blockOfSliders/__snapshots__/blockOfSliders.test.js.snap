// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BlockOfSliders /> matches snapshot 1`] = `
<div
  className="Sliders__block"
>
  <p
    className="Sliders__block--title"
  >
    Delay
  </p>
  <div
    className="Sliders__block--sliders"
  >
    <Connect(OneSlider)
      blockName="Delay"
      effectName="feedback"
      effectValues={
        Object {
          "maxValue": 1,
          "minValue": 0,
          "step": 0.01,
          "value": 0,
        }
      }
      key="feedback"
    />
    <Connect(OneSlider)
      blockName="Delay"
      effectName="time"
      effectValues={
        Object {
          "maxValue": 5,
          "minValue": 0,
          "step": 0.001,
          "value": 0,
        }
      }
      key="time"
    />
    <Connect(OneSlider)
      blockName="Delay"
      effectName="mix"
      effectValues={
        Object {
          "maxValue": 1,
          "minValue": 0,
          "step": 0.001,
          "value": 0,
        }
      }
      key="mix"
    />
  </div>
</div>
`;
